packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -139,8 +137,12 @@ class _FakeProcess implements Process {
         }
         return _exitCode;
       }),
-      stderr = Stream<List<int>>.value(utf8.encode(_stderr)),
-      stdout = Stream<List<int>>.value(utf8.encode(_stdout));
+      stderr = _stderr == null
```

Comment:
We needed a way to differentiate the stream being empty versus containing an empty string.

Reply:
what does `utf8.encode('')` return?

Reply:
EmptyList, but the Stream constructor leads to a non-empty stream with a single empty list emitted on it.